### PR TITLE
Fix error when running a command that attempts to save config file when it does not exist

### LINKED ISSUE #245: Error when running `bazel clean` and choosing to remember my choice
## Steps
1. Be sure not to have a `$HOME/.aspect` file.
2. Change to a directory with the `aspect-cli` repository.
3. Execute `bazel clean`.

The following is the output:

```sh
✔ Reclaim disk space for this workspace (same as bazel clean)
You can skip this prompt to make 'aspect clean' behave the same as 'bazel clean'
Remember this choice and skip the prompt in the future: y
Error: failed to update config file: Config File ".aspect" Not Found in "[/Users/chuck]"
```

Without having looked at the code, I suspect that we need to create the file if it does not exist.

### PR DESCRIPTION
**Pull request by cgrindel**
Closes #245.

### CODE REVIEW

File: pkg/configutils/Write.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/*
```
Comment (alexeagle) [inline pkg/configutils/Write.go:1]: we don't have any other go sources where the filename is title case like this

File: pkg/aspect/root/flags/interceptor.go [hunk 2]
```diff
@@ -102,10 +102,12 @@ func FlagsInterceptor(streams ioutils.Streams) interceptors.Interceptor {
 			// Search for config in home directory with name ".aspect" (without extension).
```
Comment (alexeagle) [inline pkg/aspect/root/flags/interceptor.go:102]: comment now out-of-date?

File: pkg/configutils/Write.go (hunk 1 above)
Comment (cgrindel) [reply pkg/configutils/Write.go:1]: Doh.

File: pkg/aspect/root/flags/interceptor.go (hunk 2 above)
Comment (cgrindel) [reply pkg/aspect/root/flags/interceptor.go:102]: I am not fixing that in this PR. We will fix with #252.

**Review by alexeagle — APPROVED** [review]

**Pull request merged**